src/components/Toggle/index.js

```diff
@@ -4,24 +4,62 @@ import styles from './Toggle.module.scss';
 import { Text } from 'components';
 import ToggleComponent from 'react-toggle';
 import '../../styles/Toggle.scss';
+import { uniqueId } from 'lodash';
```

Comment:
@c-o-l-o-r  what's the uuid stuff for?

Reply:
I thought i was going to use it like switch was, but decided not to and forgot to remove :)